src/style.css

```
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: 0;
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  font-weight: 800;
+}
+
+.bigInput {
+  height: 70px;
+  padding: 22px 0 23px 62px;
+  font-size: 16px;
+  line-height: 21.86px;
+}
+
+.smallInput {
+  height: 42px;
+  padding: 10px 0 10px 33px;
+  font-size: 14px;
+  line-height: 19.12px;
+}
+
+.bigButton {
+  top: 25px;
+  left: 25px;
+  background: none;
+  border: 0;
+  position: absolute;
+  cursor: pointer;
+}
+
+.smallButton {
+  left: 13px;
+  top: 13px;
+  background: none;
+  border: 0;
+  position: absolute;
+  cursor: pointer;
+  padding: 0;
+}
+
+.bigButtonImage {
+  width: 19px;
+}
+
+.smallButtonImage {
+  width: 11px;
+}
+
+.bigInput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 800;
+}
+
+.smallInput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 800;
+}
+
+.bigInput:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.smallInput:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigInput::placeholder {
+  font-weight: 300;
+}
+
+.smallInput::placeholder {
+  font-weight: 300;
+}
```

[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2-nd point
```suggestion
.input::placeholder {
  font-weight: 300;
}
```